File: src/main/java/com/randomperson22/piggymodtest/entities/piggy/EntityPiggy.java
package com.randomperson22.piggymodtest.entities.piggy;

import com.randomperson22.piggymodtest.entities.EntityPiggyBase;
import com.randomperson22.piggymodtest.init.ModSounds;
import net.minecraft.util.SoundEvent;
import net.minecraft.world.World;

public class EntityPiggy extends EntityPiggyBase {
    @Override
    public SoundEvent getJumpscareSound() {
        return ModSounds.ENTITY_PIGGY_JUMPSCARE;
    }

    public EntityPiggy(World worldIn) {
        super(worldIn);
    }
}

File: src/main/kotlin/com/randomperson22/piggymodtest/entities/piggy/EntityPiggy.kt
package com.randomperson22.piggymodtest.entities.piggy

import com.randomperson22.piggymodtest.entities.EntityPiggyBase
import com.randomperson22.piggymodtest.init.ModSounds
import net.minecraft.util.SoundEvent
import net.minecraft.world.World

class EntityPiggy(worldIn: World) : EntityPiggyBase(worldIn) {
    override val jumpscareSound: SoundEvent
        get() = ModSounds.ENTITY_PIGGY_JUMPSCARE
}